src/splits/splitHelpers.ts

```
@@ -555,3 +583,294 @@ const encodeCreateOWRecipientTxData = (
+}): Promise<string> => {
+  try {
+    const chainConfig = CHAIN_CONFIGURATION[chainId];
+    if (!chainConfig?.OVM_FACTORY_ADDRESS?.address) {
+      throw new Error(`OVM Factory not configured for chain ${chainId}`);
+    }
+
+    const splitsClient = await createSplitsClient(signer, chainId);
+    const executeCalls: any[] = [];
+
+
+    if (rewardRecipients && !isRewardsSplitterDeployed) {
```

ESLint

[error] More than 1 blank line not allowed.

(no-multiple-empty-lines)